gatekeeper/src/main/java/eu/arrowhead/core/gatekeeper/quartz/RelaySuprvisor.java

```diff
@@ -0,0 +1,22 @@
+package eu.arrowhead.core.gatekeeper.quartz;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import eu.arrowhead.relay.gatekeeper.GatekeeperRelayClient;
+
+public class RelaySuprvisor {
+
+	//=================================================================================================
+	// members
+	
+	private final static Set<GatekeeperRelayClient> RELAY_CLIENTS = new HashSet<>();
```

Comment:
I don't see exactly how to use this, but if you use in multiple threads, that could cause a problem, because HashSet is not thread safe.

Maybe use Collections.synchronizedSet(new HashSet<>()) instead just to be sure.